yarn.lock

```diff
@@ -2,7 +2,7 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.5.5", "@babel/code-frame@^7.8.3":
+"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.8.3":
```

Comment:
Can you reset this files changes? Seems unrelated to the other changes here...

Reply:
FWIW I think this just happened when I ran `yarn` locally`, unsure. But yes, will reset